experimental/casemapping/src/error.rs

```diff
@@ -0,0 +1,40 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use core::char::DecodeUtf16Error;
+use displaydoc::Display;
+use icu_codepointtrie::error::Error as CodePointTrieError;
+
+/// A list of possible errors for the [`CaseMapping`](crate::CaseMapping) struct
+#[derive(Display, Debug, PartialEq)]
```

Comment:
Request: make this `Copy` if possible.  #1396

Reply:
DecodeUTF16Error and CodePointTrieError are not `Copy`, and the latter is not yet `Clone`. Easy enough to make both of our own errors `Clone`, at least.

Reply:
Discussion: https://github.com/unicode-org/icu4x/issues/1396#issuecomment-1035509876